decoimpact/business/entities/rules/axis_filter_rule.py

```diff
@@ -21,34 +21,30 @@
 
 
 class AxisFilterRule(RuleBase, IArrayBasedRule):
-
-    """Implementation for the layer filter rule"""
+    """Implementation for the axis filter rule"""
 
     def __init__(
         self,
         name: str,
         input_variable_names: List[str],
-        layer_number: int,
+        element_index: int,
         axis_name: str,
-        output_variable_name: str = "output",
-        description: str = "",
     ):
-        super().__init__(name, input_variable_names, output_variable_name, description)
-        self._layer_number = layer_number
+        super().__init__(name, input_variable_names)
+        self._element_index = element_index
         self._axis_name = axis_name
 
```

Comment:
Hm, is "element_index" the proper name for the index of an axis?

Reply:
It is allot better than layer_number. This was probably a copy paste of layer_filter_rule